include/revng/Support/Debug.h

```diff
@@ -23,6 +23,13 @@ extern size_t MaxLoggerNameLength;
 
 #define debug_function __attribute__((used, noinline))
 
+/// \brief Emits Indents "\t" to the provided stream
+template<typename OStream>
+void indent(OStream &OS, size_t Indents) {
+  for (size_t I = 0; I < Indents; I++)
+    OS << "\t";
```

Comment:
Have you seen `LoggerIndent`?
Also, we usually use two white spaces to indent debug output. Let's not use tabs for the sake of coherence.

Reply:
that works only if it's a llvm::ostream and not a std::otstream.
and if we accept only llvm::ostream at that point there would not be any need to OS to be a template

Reply:
OK, then just switch to spaces.
One day we'll move to LLVM's streams. Not today.